src/components/Task.jsx

```diff
@@ -1,6 +1,10 @@
 import { Button, Stack, Modal } from "react-bootstrap";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faCheck, faPenToSquare, faTrash } from "@fortawesome/free-solid-svg-icons";
+import {
```

Comment:
Also here you are again formatting the code, which we don't want to be doing.
The commits should only be solving the issue that you have been assigned to.